databricks/sdk/core.py

```diff
@@ -491,6 +491,8 @@ class Config:
     metadata_service_url = ConfigAttribute(env='DATABRICKS_METADATA_SERVICE_URL',
                                            auth='metadata-service',
                                            sensitive=True)
+    connection_pool_size: int = ConfigAttribute()
+    connection_pool_max_size: int = ConfigAttribute()
```

Comment:
Here's my suggestion for the naming:
* `max_connection_pools`: maximum number of connection pools used by the HttpAdapter.
* `max_connections_per_pool`: maximum number of connections per pool used by the HttpAdapter.

Reply:
sure. let's get it shipped